### PR TITLE
Fixed some typos

### PR DESCRIPTION
**Pull request by Himanshunitrr**

### CODE REVIEW

File: content/en/docs/gsoc/_index.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ Ideas
 
 ### Media sharing
 
-One of the nice features of Subsonic is its [ability to generate a sharing link](http://www.subsonic.org/pages/features.jsp) for a track/album/artist/playlist to send to friends, so that they can listen/download the music without having an account on your instance. This is a nice replacement to alternative to youtube links to share music.
+One of the nice features of Subsonic is its [ability to generate a sharing link](http://www.subsonic.org/pages/features.jsp) for a track/album/artist/playlist to send to friends, so that they can listen/download the music without having an account on your instance. This is a nice replacement to an alternative to youtube links to share music.
```
Comment (deluan) [inline content/en/docs/gsoc/_index.md:76]: I think @jvoisin meant "This is a nice alternative to youtube links to share music." here

**Review by deluan — COMMENTED** [review]
Thanks for this. Just want @jvoisin to also review it before merging it.

File: content/en/docs/gsoc/_index.md (hunk 1 above)
Comment (Himanshunitrr) [reply content/en/docs/gsoc/_index.md:76]: Yeah, I also had the same thought.

Comment (jvoisin) [reply content/en/docs/gsoc/_index.md:76]: Yup, I confirm. Please change it to something like `This is a nice replacement for youtube links to share music.`

File: content/en/docs/gsoc/_index.md [hunk 2]
```diff
@@ -132,12 +132,12 @@ This task can be nicely paired with the jukebox one.
...
 
 - Implementing the glue between Alexa/Google home and the Subsonic API
-- Implementing the frontend part to configure this machinerie
+- Implementing the frontend part to configure this machinery
 
 #### Details
 
 * Skill level: Hard
-* Required abilities: Go, having an Alexa/Google Home is a plus, but worse case we can send the student a device.
+* Required abilities: Go, having an Alexa/Google Home is a plus, but in the worse case we can send the student a device.
```
Comment (jvoisin) [inline content/en/docs/gsoc/_index.md:140]: No need to add `in the` here.

**Review by jvoisin — CHANGES_REQUESTED** [review]

File: content/en/docs/gsoc/_index.md (hunk 2 above)
Comment (deluan) [reply content/en/docs/gsoc/_index.md:140]: @Himanshunitrr Is this resolved?

Comment (Himanshunitrr) [reply content/en/docs/gsoc/_index.md:140]: According to me ```in the``` made more sense but I am not expert in english so @jvoisin must be right. So it's resolved.

Comment (deluan) [reply content/en/docs/gsoc/_index.md:140]: Ok, but you need to remove it from your PR. There's also the other change requested. Please change these two things so I can merge it. Thanks!

(By the way, nobody here is expert in English :P)

**Review by jvoisin — APPROVED** [review]

**Pull request merged**